### ISSUE TITLE
Support for stripeAccount parameter

### ISSUE THREAD

**Issue by Bibendus83** [ectoflow/vue-stripe-elements#116]
It seems this library does not support the stripeAccount parameter for stripe connect accounts, am I correct?
Will it be possible to implement it?
Here is the documentation about it
https://stripe.com/docs/js/initializing#init_stripe_js-options-stripeAccount
In total there are 3 options that should be added: `stripeAccount`, `apiVersion`, and `locale`

**Comment by Bibendus83** [comment]
That's a duplicate of #106, however I could not find the issue with search due to the parameter StripeAccount written with the capital S.

**Comment by softbeehive** [comment]
Hi, I've just published v1.0.0, you can pass it with `instanceOptions`
```vue
<StripeElements
      :instance-options="instanceOptions"
      :stripe-key="stripeKey"
      ...
>
...
</StripeElements>
```

Check [examples](https://github.com/ectoflow/vue-stripe-elements#3-use-built-in-components)
